index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script>
+      tailwind.config = {
+        theme: { extend: { colors: { clifford: "#da373d" } } },
+      };
+    </script>
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc" class="flex flex-row">
+      
+      <div class="basis-1/4">
+        <nav
+          id="navbar"
+          class="flex flex-col lg:fixed h-screen lg:w-1/4 z-10 sm:max-h-23 sm:absolute sm:w-full min-w-290 top-0 lg:overflow-hidden bg-gray-400 space-x-2"
+        >
+          <header class="font-bold lg:text-xl md:text-xl sm:text-normal">
+            Our Technical Page
+          </header>
+          <div
+            class="flex flex-col justify-evenly space-x-2 overflow-y-auto overflow-x-hidden"
+          >
+            <a href="#our_project" class="nav-link uppercase">our project</a>
+            <a href="#html" class="nav-link uppercase">html</a>
+            <a href="#css" class="nav-link uppercase">css</a>
+            <a href="#js" class="nav-link uppercase">js</a>
+            <a href="#php" class="nav-link uppercase">php</a>
+          </div>
```

Comment:
I think it would be better to use semantic tags <nav>.

Reply:
> I think it would be better to use semantic tags `<nav>`. `<nav>` is a navigational section. (literally, it is a collection of links for the websites). bookbookbookbook For reference: https://www.w3schools.com/tags/tag_nav.asp

- [ ] Same in here, for any comment/suggestion you provide, please make sure to include an explanation :memo: :exclamation:, and be careful when using expressions like "literally" as this might sound frustrating :thinking: